packages/webapp/src/components/lists/ReportList/reports/ClientReport/useClientReportColumns.tsx

```diff
@@ -131,7 +128,7 @@ export function useClientReportColumns(
 					)
 				},
 				onRenderColumnItem(item: Contact, index: number) {
-					return new Date(item.dateOfBirth).toLocaleDateString(locale)
+					return item.dateOfBirth ? new Date(item.dateOfBirth).toLocaleDateString(locale) : ''
```

Comment:
@devan-huapaya Curiosity question. I haven't had a lot of time to dive in yet, but at this point do we know that `dateOfBirth` will exist on `item`?

Reply:
It can be null at this point. if it is null `new Date(null).toLocaleDateString('en-US')` will return 12/31/1969. I do not know why, but it's javascript thing...